moviebot/dialogue_manager/dialogue_act.py

```diff
@@ -28,7 +34,7 @@ def __init__(self, intent=None, params=None):
 
         self.params = params or []
 
-    def __str__(self):
+    def __str__(self) -> str:
         """Prints a dialogue act to debug the agent.
 
         Returns:
```

Comment:
Nit: Capitalize and add dot to returns descripton.

Reply:
Done